### ISSUE TITLE
add light mode support

### ISSUE THREAD

**Issue by matannahmani** [Illyism/gradient-picker#2]
would it be possible to add light mode support for https://gradient.page/hex/1F1C2C ?

**Comment by Illyism** [comment]
> would it be possible to add light mode support for https://gradient.page/hex/1F1C2C ?

Yeah, it will be eventually possible on https://gradient.page/tools/shadcn-ui-theme-generator

Hang tight!
